weio/bladed_out_file.py

```diff
@@ -0,0 +1,288 @@
+# -*- coding: utf-8 -*-
+import os
+import numpy as np
+import re
+import pandas as pd
+
+class Read_bladed_file:
```

Comment:
Can you rename the class `BladedFile ` to be in harmony with the other file formats?

Reply:
Roger that